packages/components/src/utils/validation/validateProps.ts

```diff
@@ -78,7 +78,7 @@ export const getBreakpointCustomizableStructure = <T>(
   allowedValues: Exclude<AllowedTypeKey, 'string'> | T[] | readonly T[]
 ): string => {
   if (allowedValues !== 'boolean' && allowedValues !== 'number') {
-    allowedValues = formatArrayOutput(allowedValues)
+    allowedValues = formatArrayOutput(allowedValues.map((val) => (typeof val === 'function' ? val.name : val)))
```

Comment:
I am not so sure about all these changes and added complexity. Is this really necessary?  
Or why didn't we need them in other scenarios, e.g.
```
  slidesPerPage: AllowedTypes.oneOf<ValidatorFunction>([
    AllowedTypes.breakpoint('number'),
    AllowedTypes.oneOf(['auto']),
  ]),
```

or
```
value: AllowedTypes.oneOf<ValidatorFunction>([AllowedTypes.string, AllowedTypes.number]),
```

or 
```
value: AllowedTypes.oneOf<ValidatorFunction>([AllowedTypes.string, AllowedTypes.number]),
```

Reply:
'column' should be breakpoint customisable -> e.g. `<p-segmented-control columns="{base: '2', s: 4, l: 'auto'}"/>`.  'slidesPerPage' can not have the value 'auto' within breakpoint customisable. It is only possible as single value.
![image](https://github.com/porsche-design-system/porsche-design-system/assets/109523972/442b13e5-ac8b-47a0-9150-03987069c8f2). The other examples do also not support 'breakpoint' but 'oneOf'.